cookbooks/aws-parallelcluster-slurm/recipes/update/update_head_node.rb

```diff
@@ -201,7 +202,30 @@ def update_nodes_in_queue(strategy, queues)
   only_if { ::File.exist?(node['cluster']['previous_cluster_config_path']) && is_slurm_database_updated? }
 end unless on_docker?
 
-# Update rotation script to update secret arn
+# Update check login nodes status script to update pool name
+template "#{node['cluster']['scripts_dir']}/slurm/check_login_nodes_stopped.sh" do
+  source 'slurm/head_node/check_login_nodes_stopped.sh.erb'
+  owner 'root'
+  group 'root'
+  mode '0700'
+  variables(
+    cluster_name: node['cluster']['cluster_name'] || node['cluster']['stack_name'],
+    login_nodes_pool_name: lazy { node['cluster']['config'].dig(:LoginNodes, :Pools, 0, :Name) },
+    region: node['cluster']['region']
+  )
+  only_if do
+    node['cluster']['config'].dig(:LoginNodes) &&
+      ::File.exist?(node['cluster']['previous_cluster_config_path']) &&
+      is_login_nodes_pool_name_updated?
```

Comment:
If you move this block to a sub-recipe, you should be able to add an if clause with these two conditions outside the `include_recipe` only here in `update_head_node.rb`.

Reply:
Done!